compiler/resolution/functionResolution.cpp

```diff
@@ -4705,7 +4705,14 @@ static bool isFieldAccess(CallExpr* callExpr);
 *                                                                             *
 * If this is not the case, then we will expand the call into the normal field *
 * initialization.                                                             *
-*                                                                             *
+*
+* concern for default inits:
+*
+*   make default sync (in default argument handling)
+*   =
+*   deadlock b/c RHS is empty
+*
```

Comment:
The above comment is a bit cryptic. If important, please clarify it. Otherwise remove altogether?